pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -63,7 +63,7 @@ func (pl *DefaultPreemption) Name() string {
 }
 
 // New initializes a new plugin and returns it.
-func New(dpArgs runtime.Object, fh framework.Handle, fts feature.Features) (framework.Plugin, error) {
+func New(_ context.Context, dpArgs runtime.Object, fh framework.Handle, fts feature.Features) (framework.Plugin, error) {
```

Comment:
since we haven't need a context so far, can we just pass a logger?

Reply:
Of course, just to be more general, I'll change it to a pass logger.

Reply:
hmm, I really think it is unwise to assume all use of context is for logger. it is used for tracing, context.Done, all sorts of things. The more things converted from ctx -> logger, more things we have to convert back for extensions.

Reply:
We initialized with New() for only once here for plugins, so I think it's ok to pass context, but we should use logger when processing scheduling logics. make sense?

Reply:
+1
Given the needs for wasm, I'm ok passing a context in this case.